ui/desktop/app/styles/app.scss

```diff
@@ -506,6 +506,7 @@ $chevron-down: url("data:image/svg+xml;utf-8,<svg viewBox='0 0 24 24' fill='%231
   }
 }
 
+.session-details-header,
```

Comment:
None blocking comment:
session-detail-header and target-details-header are basically the same, we just add the domain (`session-` or `target-`) but the code is the  same, the design is the same, isn't?
Do you think it is doable to refactor this something more reusable? perhaps defining this such as:  `.rose-layout-page-header > h1` or something that does not add the domain name to the class?

Reply:
That is exactly what I had planned. I don't believe these page headers will require different styles but I was keeping it split on purpose for now. But the plan is to look at the styling for both pages and see what I can refactor into a single style block that can be used by both screens.